### PR TITLE
Fill

### PR DESCRIPTION
**Pull request by gintama91**

### CODE REVIEW

File: lib/scarpe/wv/fill.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+class Scarpe
+  class WebviewFill < WebviewWidget
+    def initialize(properties)
+      super(properties)
+    end
+
+    def element
+      width = @parent.get_style[:width]
+      height = @parent.get_style[:height]
+      puts "width is #{width}"
```
Comment (noahgibbs) [inline lib/scarpe/wv/fill.rb:12]: Should remove this puts

**Review by Schwad — COMMENTED** [review]
Nice. Can you do an example with a couple different stacks and a couple different fills? 

And include the screenshot in your description?

Thanks for the PR looking good!

**Comment by gintama91** [comment]
okay i will change the things suggested

**Comment by gintama91** [comment]
image

**Comment by gintama91** [comment]
> okay i will change the things suggested

Done

**Review by noahgibbs — APPROVED** [review]

**Pull request merged**